### DISCUSSION TITLE
Idea: Improving FDM print quality by keeping print times low with optimized outer layer heights

### DISCUSSION THREAD

**Discussion by chropo8** [OrcaSlicer/OrcaSlicer#7598] [Ideas]
I have an idea for making FDM prints look better without significantly increasing print time.

The concept is to divide only the outer contour in height and print this outer contour in multiple steps with 1/2 or 1/3 layer thickness.
The inner contours and structures can be printed with regular layer thickness to save time.
To make this work, the print order would likely need to be set to Outer/Inner, but that shouldn't be a problem—except possibly for critical overhangs.

_For example: For a cube with a 20mm edge length, the print time increases by 84% when halving the layer thickness. With the solution described above, the print time would only increase by 23%—but the exterior would look just as good as the one that took 84% longer. A significant time advantage._

Maybe somebody has already tried something like this?
I'm not a developer - just a 3d printing enthusiast. 
So from my own skills I wouldn't be able to bring that idea to a machine. 

**But maybe there is somebody out there who can!**

**Comment by xantrk** [comment]
Sounds like the opposite of "Combine Infill Every N Layers" option, but you want to combine inner walls. I think this is a great idea!
